tests/cross/src/test/scala/tests/pc/CompletionMatchSuite.scala

```diff
@@ -103,6 +103,33 @@ object CompletionMatchSuite extends BaseCompletionSuite {
     filter = _.contains("exhaustive")
   )
 
+  checkEdit(
+    "Sort auto-completed exhaustive match keywords by declaration order",
```

Comment:
we generally use short meaningful names for the tests (e.g. `exhaustive-sorting`), so that they can be used as targets for `testOnly`.